src/index.ts

```diff
@@ -64,49 +66,57 @@ let answers: Answers = {};
 
 const program = new Command(packageJson.name)
     .version(packageJson.version, '-v, --version')
-    .option('-t, --type <type>', 'Solution Type: remotemonitoring', /^(remotemonitoring|test)$/i, 'remotemonitoring')
-    .option('-s, --sku <sku>', 'SKU Type: basic, standard, or test', /^(basic|standard|test)$/i, 'basic')
+    .option('-t, --type <type>', 'Solution Type: remotemonitoring, devicesimulation, devicesimulation-nohub',
```

Comment:
Just a question about this design. if we define flatten solution type here, it will not be consistent with the solution tiles on IoTSuite site customer can see. is it better to provide an new option for optional resource provisioning in the same solution type 'devicesimulation'?  The script and template have duplication and hard to maintain together because you need to keep both of them updated once requirement change in the future.

Reply:
I agree this is not an ideal situation, in fact the work done here it purely to enable development, and I don't think we will merge it to master in this state. There is a lot of work to do in the CLI to decouple it from assumptions taken during the last 6 months, and generally speaking a lot of the work to do is dependent on the Suite design we've discussed offline.